CHANGELOG.md

```diff
@@ -28,7 +31,7 @@ _None_
 
 * Improved installation instructions in the README.  
   [Olivier Halligon](https://github.com/alisoftware)
-  [#303](https://github.com/SwiftGen/SwiftGenKit/issues/303)
+  [#303](https://github.com/SwiftGen/SwiftGen/issues/303)
```

Comment:
We should definitely add a _Danger_ rule to check if the links exist and if it points to SwiftGen (e.g. maybe all links associated with each SwiftGen's CHANGELOG entries should point to an issue or PR in the SwiftGen repo not the other repos… except for the PR before a release when we might reference big changes from other repos and ignore the future Danger warning then)